packages/@n8n/config/test/decorators.test.ts

```diff
@@ -0,0 +1,19 @@
+import { Container } from 'typedi';
+import { Config, Env } from '../src/decorators';
+
+describe('decorators', () => {
+	beforeEach(() => {
+		Container.reset();
+	});
+
+	it('should throw when explicit typing is missing', () => {
+		expect(() => {
+			@Config
+			class InvalidConfig {
+				@Env('STRING_VALUE')
+				value = 'string';
+			}
+			Container.get(InvalidConfig);
+		}).toThrowError('Invalid decorator metadata on key "value" on InvalidConfig');
```

Comment:
The thrown error message is not very actionable. What's the fix here? Could we include it in the error message